mods/page_revisions.js

```diff
@@ -68,6 +70,7 @@ PRS.prototype.getTableSchema = function () {
                 { attribute: 'rev', type: 'hash' },
                 { attribute: 'tid', type: 'range', order: 'desc' },
                 { attribute: 'title', type: 'range', order: 'asc' },
+                { attribute: 'hidden', type: 'proj' },
                 { attribute: 'tags', type: 'proj' }
```

Comment:
We can probably drop tags from the projection if we are using 'hidden' for acls.